lua/entities/gmod_wire_pod.lua

```diff
@@ -442,6 +442,10 @@ function ENT:Think()
 						button.EntToOutput = ply
 						button:Switch( true )
 					end
+				elseif button.Use then
```

Comment:
This clause is the same as the `gmod_wire_lever` clause above.

Reply:
I think that the button clause could be the same, but additionally set `button.podpress`.